client/src/components/pages/Players.js

```diff
@@ -25,6 +25,8 @@ const { Content } = Layout;
 const { Panel } = Collapse;
 const { Option } = Select;
 
+let prevPlayers = null;
```

Comment:
is this used? i don't see `prevPlayers` be assigned a value anywhere

Reply:
This was supposed to be a guard against unnecessary recalculations (passing player as a dep to useEffect doesn't seem to work), but apparently I forgot to set it after

Reply:
Fixed in 28cbdf7f7d1d074f853160bcff8c0669ac6a7fc6